src/it/java/io/deephaven/benchmark/tests/standard/updateby/CumMaxTest.java

```diff
@@ -0,0 +1,43 @@
+/* Copyright (c) 2022-2023 Deephaven Data Labs and Patent Pending */
+package io.deephaven.benchmark.tests.standard.updateby;
+
+import org.junit.jupiter.api.*;
+import io.deephaven.benchmark.tests.standard.StandardTestRunner;
+
+/**
+ * Standard tests for the updateBy table operation. Calculates a cumulative maximum for specified columns and places the
+ * result into a new column for each row.
+ */
+public class CumMaxTest {
+    final StandardTestRunner runner = new StandardTestRunner(this);
+
+    @BeforeEach
+    public void setup() {
+        runner.api().table("source").random()
+                .add("int5", "int", "[1-5]")
+                .add("int10", "int", "[1-10]")
+                .add("str100", "string", "s[1-100]")
+                .add("str150", "string", "[1-150]s")
+                .generateParquet();
+        runner.api().query("from deephaven.updateby import cum_max").execute();
+    }
+
+    @Test
+    public void cumMax1Group2Cols() {
```

Comment:
This is a general comment for all the UpdateBy tests.  It is perfectly valid to have zero grouping columns (this is the `Zero-Key` case to which we often refer).  I would like to see this tested as well for each of the operators.  Something like this:

```
    @Test
    public void cumMax0Group3Cols() {
        var q = "source.update_by(ops=cum_max(cols=['X=int5','CP2=int10']))";
        runner.test("CumMax- 0 Group 100 Unique Vals 3 Cols", runner.scaleRowCount, q, "str100", "int5", "int10");
    }
```